app/javascript/packs/MainApp/components/CertificationContent.jsx

```diff
@@ -23,6 +25,24 @@ const certificationLinkConfig = {
   }
 }
 
+const oldCertificationLinkConfig = {
+  snitch: {
+    title: 'Snitch Referee Written Test',
```

Comment:
We should add “16–18” behind every test title so referees know what they will be tested.